xbmc/Application.h

```diff
@@ -142,6 +142,7 @@ friend class CAppInboundProtocol;
   bool Initialize() override;
   void FrameMove(bool processEvents, bool processGUI = true) override;
   void Render() override;
+  bool IsRenderGuiEnabled() override { return m_renderGUI; }
```

Comment:
no need introduce new method, use `GetRenderGUI`

Reply:
Missed that one. DaveTBlake I guess you can take care of this?

Reply:
Yes I would hope so,